prov/ansible/execution.go

```diff
@@ -573,52 +573,95 @@ func (e *executionCommon) resolveOperationOutputPath() error {
 	e.HaveOutput = true
 	//We iterate over all entity of the output in this operation
 	for outputName, outputValue := range mapOutputs {
-		va := outputValue.ValueAssign
-		if va.Type != tosca.ValueAssignmentFunction {
-			return errors.Errorf("Output %q for operation %v is not a valid get_operation_output TOSCA function", outputName, e.operation)
-		}
-		oof := va.GetFunction()
-		if oof.Operator != tosca.GetOperationOutputOperator {
-			return errors.Errorf("Output %q for operation %v (%v) is not a valid get_operation_output TOSCA function", outputName, e.operation, oof)
-		}
-		targetContext := oof.Operands[0].String() == "TARGET"
-		sourceContext := oof.Operands[0].String() == "SOURCE"
-		if (targetContext || sourceContext) && !e.operation.RelOp.IsRelationshipOperation {
-			return errors.Errorf("Can't resolve an operation output in SOURCE or TARGET context without a relationship operation: %q", va.String())
+		// either output can be associated to a get_operation_output value assignment or to an attribute mapping
+		if outputValue.ValueAssign != nil {
```

Comment:
Not in this PR but we should definitively think about extracting this from the ansible package!

Reply:
Yes !